packages/opentelemetry-node/docs/intro.asciidoc

```diff
@@ -0,0 +1,52 @@
+////
+Goal of this doc:
+Provide all the information a user needs to determine if the product is a good enough fit for their use case to merit further exploration
+
+Assumptions we're comfortable making about the reader:
+* They are familiar with Elastic
+* They are familiar with OpenTelemetry
+////
+
+[[intro]]
+== Elastic Distribution for OpenTelemetry Node.js
+
+++++
+<titleabbrev>Introduction</titleabbrev>
+++++
+
+:language: Node.js
+:distro_name: Elastic Distribution for OpenTelemetry {language}
+
+include::release-status.asciidoc[]
+
+////
+✅ Intro
+////
+The {distro_name} ("the distro") is a {language} package that provides:
+
+* An easy way to instrument your application with OpenTelemetry.
+* Configuration defaults for best usage.
+
+////
+✅ What is it?
+✅ Why use it?
+////
+A _distribution_ is a wrapper around an upstream OpenTelemetry repository with some customizations. The {distro_name} is a wrapper around the https://opentelemetry.io/docs/languages/js[OpenTelemetry SDK for {language}] and includes the following customizations:
+
+* Uses Elastic-specific processors that ensure optimal compatibility when exporting OpenTelemetry signal data to an Elastic backend like Elastic APM server.
+* Preconfigures the collection of tracing and metrics signals, applying some opinionated defaults, such as which sources are collected by default.
+* Ensures that the OpenTelemetry protocol (OTLP) exporter is enabled by default.
+
+NOTE: For more details about OpenTelemetry distributions in general, visit the https://opentelemetry.io/docs/concepts/distributions[OpenTelemetry documentation].
+
+////
+✅ How to use it?
+////
+Use the distro to start the OpenTelemetry SDK with your {language} application to automatically capture tracing data, performance metrics, and logs. Traces, metrics, and logs are sent to any OTLP collector you choose.
+
+Start with helpful defaults to begin collecting and exporting OpenTelemetry signals quickly. Then, further refine how you use the distro using extension methods that allow you to fully control the creation of the underlying tracer and metric providers.
+
+After you start sending data to Elastic, use an {observability-guide}/index.html[Elastic Observability] deployment -- hosted on Elastic Cloud or on-premises -- to monitor your applications, create alerts, and quickly identify root causes of service issues.
+
```

Comment:
Will this "intro" page be a landing page for incoming links? If so, it would be nice to prominently link to the get-started page, and perhaps other doc pages.

Reply:
>Will this "intro" page be a landing page for incoming links?

It depends on the links. 🙂  

>it would be nice to prominently link to the get-started page, and perhaps other doc pages.

I'm hesitant to add too many links, which (especially with such a small doc set) would be like recreating the table of contents, but we could definitely encourage them to go to the get started doc next.

Reply:
> but we could definitely encourage them to go to the get started doc next.

Sounds good. I'm trying not to micro-manage ... tooooo  much. :)

Reply:
>Sounds good. I'm trying not to micro-manage ... tooooo much. :)

I appreciate your input! _Especially_ because I know I'm coming into the process pretty late.